### ISSUE TITLE
[iOS] Issue building the app if the `scheme` contains a space

### ISSUE THREAD

**Issue by thep-dr** [FormidableLabs/react-native-owl#148]
Hi FormidableLabs,

Thanks for a great tool 🥳 

I have trouble building my app using owl if there is a space in the `scheme` (`npx owl build --platform ios`).

I have the following `owl.config.json`:
```
  "ios": {
    "workspace": "ios/test-app.xcworkspace",
    "scheme": "Test App",
    "configuration": "DebugProd",
    "device": "iPhone 13 Pro"
  },
```

I have tried to espace the `scheme` quotation marks, but no luck.

The build command fails with the following issue:
```
[OWL - CLI] Building the app on ios platform.
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace ios/test-app.xcworkspace -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath ios/build

User defaults from command line:
    IDEDerivedDataPathOverride = /Users/thep/Documents/nyhedsapp/ios/build
    IDEPackageSupportUseBuiltinSCM = YES

Build settings from command line:
    SDKROOT = iphonesimulator16.2

2023-04-18 15:17:02.622 xcodebuild[22526:146612] Writing error result bundle to /var/folders/5t/jbpfqfj57gbgknydy00l0hxcq_5fst/T/ResultBundle_2023-18-04_15-17-0002.xcresult
xcodebuild: error: Unknown build action 'App'.
Error: Command failed with exit code 65: xcodebuild -workspace ios/test-app.xcworkspace -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath ios/build
    at makeError (/Users/thep/Documents/nyhedsapp/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/thep/Documents/nyhedsapp/node_modules/execa/index.js:118:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  shortMessage: 'Command failed with exit code 65: xcodebuild -workspace ios/test-app.xcworkspace -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath ios/build',
  command: 'xcodebuild -workspace ios/test-app.xcworkspace -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath ios/build',
  escapedCommand: 'xcodebuild -workspace "ios/test-app.xcworkspace" -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath "ios/build"',
  exitCode: 65,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}

```

**Comment by robwalkerco** [comment]
Thanks for reporting the issue @thep-dr 

The command that is generated from your example is
`/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace ios/test-app.xcworkspace -scheme Test App -configuration DebugProd -sdk iphonesimulator -derivedDataPath ios/build` 

I can see that the value for `-scheme` needs to be quoted so that it is treated as a single string. 

Feel free to open a PR if you have a chance to, otherwise I'll look at resolving it when I have some time.

**Comment by robwalkerco** [comment]
The fix for this issue has been released in [v1.2.3](https://github.com/FormidableLabs/react-native-owl/releases/tag/v1.2.3)

**Comment by thep-dr** [comment]
Thank you so much @robwalkerco 🚀
